src/pandablocks_ioc/_tables.py

```diff
@@ -491,9 +504,21 @@ def update_table(self, new_values: List[str]) -> None:
             new_values: The list of new values from the PandA
         """
 
-        curr_mode = TableModeEnum(self.mode_record_info.record.get())
+        if self._sent_data == new_values:
+            # Received changes back from the panda that were updated
+            # from this method already
+            return
+
```

Comment:
We need to discard the sent data now so that if the table updates without us changing it we don't drop an update by accident:
```suggestion
        self._sent_data = []
```

Reply:
Please can we have a test that:
- Sends table1
- Receives table1
- assert table == table1
- Receives table2
- assert table == table2
- Receives table1
- assert table == table1

I think without this change the last assert will fail as the table won't be changed back to table1

Reply:
This did fail without that change :pray: